hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java

```diff
@@ -114,6 +113,7 @@ private void initializeAndProcessMessage() {
         if (!node.joined()) {
             throw new HazelcastInstanceNotActiveException("Hazelcast instance is not ready yet!");
         }
+        parameters = decodeClientMessage(clientMessage);
```

Comment:
@pveentjer 
This method is used by all MessageTask classes. It will mean that touching all of them. If you still think that we should rename it, I would like to do that as separate pr, right after merging this one. What do you think ?